fuse/cameron_so.py

```diff
@@ -40,7 +35,9 @@ def fuse_events(post):
 
 if __name__ == "__main__":
     cprr = pd.read_csv(data_file_path("clean/cprr_cameron_so_2020.csv"))
-    post = prepare_post_data()
+    agency = cprr.agency[0]
+    post = pd.read_csv(data_file_path("clean/pprr_post_2020_11_06.csv"))
+    post = post.loc[post.agency == agency]
```

Comment:
This is fine but what would happen if there is a typo in the agency? We should raise an error if there's a mismatch in agency text also:
```python
if len(post) == 0:
    raise ValueError("agency not found", agency)
```

Reply:
Yes. I'll make those changes now.

Reply:
and to avoid repeating code we could just put all of this in a function with name like `lib.post.load_for_agency` or something:
```python
def load_for_agency(agency):
    post = pd.read_csv(data_file_path("clean/pprr_post_2020_11_06.csv"))
    post = post.loc[post.agency == agency]
    if len(post) == 0:
        raise ValueError("agency not found", agency)
    return post
```
Then you can call it like this:
```python
from lib import post
...
post_data = post.load_for_agency(agency)
```

Reply:
Thanks